example.config.yml

```diff
@@ -1,64 +1,78 @@
 version: 1
 cloud:
-  provider: aws-ecs
+  provider: aws-ecs # Are you thinking there might be multiple provider types on top of AWS? Like aws-ec2 or something?
   ecs_cluster_name: singer
   bucket_name: some-bucket
+  docker_base_image: image:tag # optional; will use this base image or default to python:3.8 if not provided
```

Comment:
These are all not needed, they are handled internally, the user doesn't need to define them.